src/collocateElement.utility.js

```diff
@@ -381,16 +411,37 @@ const applyCoordinatesToElement = (element, coordinates) => {
 };
 
 const applyOverflowToElementContent = (element, height) => {
-  if (!element || !element.style) { return; }
+  if (!element || !element.style || !height) { return; }
   element.style.height = `${height}px`;
   element.style.overflowY = 'auto';
 };
 
+const resetElementPosition = (element) => {
+  if (!element || !element.style) { return; }
+  element.style.top = 'unset';
+  element.style.left = 'unset';
+  element.style.right = 'unset';
+  element.style.bottom = 'unset';
+  element.style.width = 'inherit';
+  element.style.maxWidth = 'inherit';
+};
+
 const resetElementContentHeight = (element) => {
   if (!element || !element.style) { return; }
   element.style.height = 'auto';
 };
 
+const applyTouchScreensCoordinates = (element, elementContent, coordinates) => {
+  if (!element || !element.style) { return; }
+  element.style.top = `${coordinates.top}px`;
+  element.style.left = `${coordinates.left}px`;
+  element.style.right = `${coordinates.right}px`;
+  element.style.bottom = `${coordinates.bottom}px`;
+  element.style.width = 'auto';
+  element.style.maxWidth = 'unset';
```

Comment:
Depends on what you are trying to do but undoing value is possible to perform via this:
```suggestion
  element.style.maxWidth = '';
```
This will unset value from the style object entirely and won't be presented as inline style at all.


**style.maxWidth = 'unset':**
```html
<div style="width: 300px; max-width: 'unset'">...
```

**style.maxWidth = '':**
```html
<div style="width: 300px; ">...
```